packer/mongo.json

```diff
@@ -17,9 +17,9 @@
         "ssh_username": "admin",
         "ami_name": "cyhy-mongo-hvm-{{timestamp}}-x86_64-ebs",
         "ami_regions": [
-            "us-east-1",
-            "us-west-1",
-            "us-west-2"
+              "us-east-1",
```

Comment:
Please remove these unnecessary spaces or just revert this file- it should not have changed at all.